functions/cli.js

```diff
@@ -1,4 +1,64 @@
 #!/usr/bin/env node
+const readline = require('readline');
+const rl = readline.createInterface(process.stdin, process.stdout);
 
-require('./lib/import/index');
+rl.question(`\nWARNING: The back fill process will index your entire collection which will impact your Search Operation Quota.  Please visit https://www.algolia.com/doc/faq/accounts-billing/how-algolia-count-records-and-operation/ for more details.  Do you want to continue? (y/N): `, function(answer) {
```

Comment:
would you be comfortable using the Promises API https://nodejs.org/api/readline.html#promises-api

Reply:
I would be comfortable with promises but its only available on Node v. 18.x.  The older versions are not have the readline set up with Promises at this time.  I will convert it over later.